src/connectors/refinement-list/connectRefinementList.js

```diff
@@ -328,12 +330,11 @@ export default function connectRefinementList(renderFn) {
           createURL,
           instantSearchInstance,
         } = renderOptions;
-        const items = results
-          .getFacetValues(attributeName, { sortBy })
-          .slice(0, this.getLimit())
-          .map(formatItems);
 
-        const hasExhaustiveItems = items.length < this.getLimit();
+        const rawItems = results.getFacetValues(attributeName, { sortBy });
```

Comment:
I wouldn't call that `rawItems` because `items` is not an unrefined state of the `items`. Why not `facetValues`?